app/views/budgets/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render "budgets/new" %>
+<div class="container d-flex flex-column justify-content-center align-items-center">
+
+  <%= form_with model: @budgets, url: group_budgets_path, method: :post, data: { turbo: false } do |f| %>
+    <div class="field mt-2 mb-2">
+        <%= f.text_field :name, autofocus: true, autocomplete: "name" , class: "form-control", placeholder: "Budget Name" %>
+    </div>
+    <div class="field mb-2">
+      <%= f.text_field :amount, autofocus: true, autocomplete: "name" , class: "form-control", placeholder: "Amount in numbers" %>
+    </div>
+
+    <div class="actions mb-2">
+      <%= f.submit "SAVE" , class: "btn btn-success" %>
+    </div>
```

Comment:
- [ ] Kindly ensure the transactions items are ordered  by the most recent 
![image](https://user-images.githubusercontent.com/61408860/211102816-b4727b81-4ad8-44eb-8b30-b13716f28b10.png)